test/core/utils/send-command-to-frame.js

```diff
@@ -13,6 +14,26 @@ describe('axe.utils.sendCommandToFrame', function() {
     assert.ok(false, 'should not be called');
   };
 
+  it('should return results from frames', function(done) {
+    var frame = document.createElement('iframe');
+    frame.addEventListener('load', function() {
+      axe.utils.sendCommandToFrame(
+        frame,
+        params,
+        function() {
+          done();
```

Comment:
I feel like you should assert the result or something rather than just saying any callback is valid